engine/table/src/main/java/io/deephaven/engine/tablelogger/EngineTableLoggerProvider.java

```diff
@@ -0,0 +1,42 @@
+package io.deephaven.engine.tablelogger;
+
+import io.deephaven.engine.tablelogger.impl.memory.EngineTableLoggerMemoryImplFactory;
+
+/**
+ * Provides the factory for providing engine table loggers.
+ */
+public class EngineTableLoggerProvider {
+    private EngineTableLoggerProvider() {
+        throw new UnsupportedOperationException();
```

Comment:
There's no need for the UOE here

Reply:
I instinctually add that in my private constructors to block against a) bad developer usage and b) reflection. I will assume you are implying Deephaven convention prefers to leave it empty, so I will remove it.